### PR TITLE
Feature/66003 attendees report changes

### PR DESCRIPTION
**Pull request by jesseeproductions**
_Ref:_ [C#66003 ](https://central.tri.be/issues/66003 )

### CODE REVIEW

File: src/template-tags/tickets.php [hunk 1]
```diff
@@ -235,6 +235,18 @@ function tribe_tickets_get_ticket_stock_message( Tribe__Tickets__Ticket_Object $
 		else {
 			$status = '';
 
+			if ( $is_global && 0 < $stock && $global_stock->is_enabled() ) {
+				$status_counts[] = esc_html( sprintf(
+					__( '%1$d Remaining of the global stock', 'ticket global stock message (remaining stock)', 'event-tickets' ),
+					(int) esc_html( $stock )
```
Comment (bordoni) [inline src/template-tags/tickets.php]: No need to `esc_html` before casting it as a `int`

File: src/template-tags/tickets.php [hunk 2]
```diff
@@ -235,6 +235,18 @@ function tribe_tickets_get_ticket_stock_message( Tribe__Tickets__Ticket_Object $
...
+				) );
+		    } else {
+				$status_counts[] = esc_html( sprintf(
+					__( '%1$d Remaining', 'ticket stock message (remaining stock)', 'event-tickets' ),
+					(int) esc_html( $stock )
```
Comment (bordoni) [inline src/template-tags/tickets.php]: _Ditto._

**Comment by jesseeproductions** [comment]
Changes made to those two points

File: src/template-tags/tickets.php [hunk 3]
```diff
@@ -221,26 +222,37 @@ function tribe_tickets_get_ticket_stock_message( Tribe__Tickets__Ticket_Object $
...
-			) );
-
-			$message = sprintf(
-				esc_html__( 'Sold %1$d (units remaining: %2$s%3$s%4$s)', 'event-tickets' ),
-				esc_html( $sold ), $stock, $cancelled_count, $pending_count
-			);
+			$status = '';
+
+			if ( $is_global && 0 < $stock && $global_stock->is_enabled() ) {
+				$status_counts[] = esc_html( sprintf( __( '%1$d Remaining of the global stock', 'ticket global stock message (remaining stock)', 'event-tickets' ), (int) $stock ) );
```
Comment (barryhughes) [inline src/template-tags/tickets.php]: Here and quite a few of the following lines, we seem to be providing context for the translations but we're using a straight `__()` - we need to fix and either drop the context or move to `_x()` or similar :+1:

**Comment by barryhughes** [comment]
:ice_skate:

**Pull request merged**